<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>basicMath</class>
 <widget class="QWidget" name="basicMath">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>274</width>
    <height>332</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="newColumnName">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>251</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>New Column Name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="firstColumn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>251</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>First Column</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="secondColumn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>251</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Second Column</string>
   </property>
  </widget>
  <widget class="QLabel" name="operationsLable">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Operations</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="sumRB">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Sum</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="subtractSB">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>190</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Subtract</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="divideSB">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Divide</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="multiplyRB">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>230</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Multiply</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>280</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Operate</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
